src/math/mod.rs

```diff
@@ -344,3 +344,97 @@ fn with_set_low_word(f: f64, lo: u32) -> f64 {
 fn combine_words(hi: u32, lo: u32) -> f64 {
     f64::from_bits((hi as u64) << 32 | lo as u64)
 }
+
+mod fdlibm {
+    /* @(#)fdlibm.h 5.1 93/09/24 */
```

Comment:
Can the items (fns and const) in this module be `pub(crate)` as opposed to just `pub` ?

Reply:
Done in #173 .